R/parse-datetime.r

```diff
@@ -11,7 +11,7 @@
 
 #' @param date contains the date to be formatted
 #' @param mode gives the desired output format (see above)
-datetime_fn <- function(createdat, mode="since") {
+datetime_fn <- function(createdat, mode="since", permissive=FALSE) {
```

Comment:
Also, if you're going to add a new parameter, you should add roxygen documentation for it. :-1: